Submitted By: Jim Gifford (jim at linuxfromscratch dot org)
Date: 2003-10-12
Initial Package Version: 1.4.2
Origin: Jim Gifford
Description: GCC 3.3 Fix for Java SDK
	     inline asm functions don't like -O3
	     This patch removes any -O3 to prevent compile errors.

	     Error message:
	     error: impossible constraint in `asm'

 
diff -Naur java.orig/hotspot/build/linux/makefiles/gcc.make java/hotspot/build/linux/makefiles/gcc.make
--- java.orig/hotspot/build/linux/makefiles/gcc.make	2003-09-11 01:40:04.000000000 +0000
+++ java/hotspot/build/linux/makefiles/gcc.make	2003-10-12 07:28:06.000000000 +0000
@@ -37,7 +37,7 @@
 CFLAGS += $(GCC_SAVE_TEMPS)
 
 # The flags to use for an Optimized g++ build
-OPT_CFLAGS += -O3 
+OPT_CFLAGS +=  
 # Hotspot uses very unstrict aliasing turn this optimization off
 OPT_CFLAGS += -fno-strict-aliasing
 
